packages/website/src/components/Curriculum/TechSliding.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { Box } from '@coderscamp/ui/components/Box';
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { useMediaQuery } from '@coderscamp/ui/hooks/useMediaQuery';
+
+import { techNamesTop } from './Curriculum.data';
+import { animation } from './TechSlidingAnimation';
+
+type TechSlidingProps = {
+  techIconsArray?: {
+    name: string;
+    value: JSX.Element;
+  }[];
+  isAnimationReverse?: 'reverse' | 'normal';
+};
+
+export const TechSliding = ({ techIconsArray = techNamesTop, isAnimationReverse = 'normal' }: TechSlidingProps) => {
+  const [isLargerThan768] = useMediaQuery('(min-width: 768px)');
```

Comment:
💭 It would be better to use `useBreakpointValue` here so animation speed change on one of our predefined breakpoints